app/helpers/application_helper.rb

```diff
@@ -136,6 +136,14 @@ def campnote_types
     ]
   end
 
+  def year_in_school
```

Comment:
Since some of the international and private schools use a system outside of 9, 10, 11 maybe adding an 'other' as a selection would be a catch-all selection.

Reply:
'Other' option is added to the grades selection.